en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,143 @@
+
+A Silverstripe CMS major release line follows a predefined life cycle. New Silverstripe CMS majors are released in odd years (e.g. 2023, 2025, 2027, etc). At any given time, only two major release lines are officially supported.
+
+### Pre-release
+
+A Silverstripe CMS stable major release is preceded by a beta period of at least three months.
+
+The purpose of the beta period is to:
+
+- allow early adopters to plan their upgrade
+- identify bugs in the major release
+- help community modules add support for the new major release early on.
+
+Once a new major enters the beta period, it should be compatible with the dependencies intended to ship in the stable release. Substantial breaking changes should be avoided post-beta.
+
+Prior to the beta period, alpha releases are published at the discretion of the development team.
+
+### Stable release
+
+New major releases of Silverstripe CMS are tagged between April and June of odd years.
+
+### Active development
+
+Once a major release is stable, it enters a period of active development. A major release in active development receives:
+- regular minor releases that ships new feature and API in a backward compatible way
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+A major releases line stays in *active development* for approximatively two years. *Active development* for a major release line ends when the next major release is tagged stable.
+
+Only one major release line is in *active development* at any given time.
+
+### Bug and security fixes
+
+A Silverstripe CMS major release line enters the *bug and security fixes* phase once the follow major release is tagged stable.
+
+A major release line in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+A major release line in the *bug and security fixes* phase does **not** receive:
+- new feature
+- new APIs
+- new minor releases.
+
+A major release line stays in the *bug and security fixes* phase for 1 year.
+
+### Security fixes only
+
+At the end of *bug and security fixes* phase, a Silverstripe CMS major release line transitions to the *security fixes only* phase.
+
+In the *security fixes only* phase, a Silverstripe CMS major release line only receives patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+### End-of-life
+
+A Silverstripe CMS major release line reaches *end-of-life* once the next major release line exits active development.
+
+End-of-life major releases do not receive updates of any kind including security fixes or bug fixes.
+
+## PHP support commitments
+
+Silverstripe CMS major releases track PHP releases. 
+
+The Silverstripe CMS release cycle is built around these assumptions.
+
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years with a year of limited support. 
```

Comment:
Should this be using the colon format? I.e. are the list items "these assumptions"?
```suggestion
The Silverstripe CMS release cycle is built around these assumptions:

- new PHP releases are shipped at the end of November on an annual basis
- PHP releases are in full support for two years with a year of limited support. 
```

Reply:
done